### DISCUSSION TITLE
公式怎么改回原先的样式

### DISCUSSION THREAD

**Discussion by TBBnozomi** [nju-lug/NJUThesis#141] [Q&A] [answered]
我原先使用的公式是默认的展示样式，就是默认空tex文件加上几个 amsmath 包的公式。现在这个模板的公式的展示效果感觉不太行，有没有办法切回原来的公式显示？

**Comment by note286** [comment]
你所述的样式指的是数学字体吗？

**Reply by TBBnozomi** [reply]
对的。这个模板的希腊字母显得有点短，分式和幂指数公式的位置感觉有点怪

**Comment by TBBnozomi** [comment, marked as answer]
我搜索到了一种解决方法。在引言区里加上 `\setmathfont{latinmodern-math.otf}` 可以解决希腊字母和分式的问题，但是会产生 `\mathbb` 使用 `bbold` 的问题。针对这个新的问题我加上了
```
\usepackage[]{amsfonts}
\DeclareSymbolFontAlphabet{\vmathbb}{AMSb}
```
这样之后目前没有发现新的问题。如果没有什么更好的办法，我会标记这个这个问题为已解决

**Reply by note286** [reply]
模板默认使用的是XITS Math，你改为使用了Latin Modern Math，和LaTeX默认的Computer Modern字体很相似，所以是你熟悉的样子。你可以阅读unicode-math的文档来了解更多用法。

@atxy-blip 可以考虑增加用户自己切换数学字体的接口，或者直接搬我写好的。

**Reply by atxy-blip** [reply]
也是打算加的，那暑假再多一个新功能吧。到时候就不客气照抄了~

**Reply by note286** [reply]
拿去抄，哈哈哈，我也从你这学到了不少:laughing:
